src/Notifications/NotificationsViewExtension.cs

```diff
@@ -117,12 +126,12 @@ internal void AddNotifications()
 
         public void Shutdown()
         {
-           // Do nothing for now
```

Comment:
Interesting
![image](https://github.com/DynamoDS/Dynamo/assets/61755417/e4f6c25e-c55a-43e2-8657-0f872aeb66b1)

Reply:
indeed:
https://github.com/DynamoDS/Dynamo/pull/11054/files

this is because as horrible as it seems, and it is, I believe Extensions have their shutdown called, but ViewExtensions have disposed called instead... 🤦 - changing it now will lead to possible double dispose and other issues, I can't find the github conversation but we've discussed various strategies not reallying coming up with anything great. This is a good thing to fix during 3.0.
@aparajit-pratap @QilongTang FYI!